### PR TITLE
[FLINK-34770][cdc][doc] fix download links.

### PR DESCRIPTION
**Pull request by lvyanquan**
Remove invalid links unify add download links to fat jar.

### CODE REVIEW

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ using a build automation tool (such as Maven or SBT) and SQL Client with SQL JAR
 
 ```Download link is available only for stable releases.```
```
Comment (loserwang1024) [inline docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md:51]: It seems redundant now, there is no unavailable link now.

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md [hunk 2]
```diff
@@ -47,9 +47,9 @@ MongoDB CDC 连接器允许从 MongoDB 读取快照数据和增量数据。 本
 
 ```下载链接仅适用于稳定版本。```
 
-下载 [flink-sql-connector-mongodb-cdc-3.0-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-connector-mongodb-cdc/3.0.0/flink-connector-mongodb-cdc-3.0.0.jar) 把它放在 `<FLINK_HOME>/lib/`.
+下载 [flink-sql-connector-mongodb-cdc-3.0.1.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mongodb-cdc/3.0.1/flink-sql-connector-mongodb-cdc-3.0.1.jar) 把它放在 `<FLINK_HOME>/lib/`.
 
-**注意:** flink-sql-connector-mongodb-cdc-XXX-SNAPSHOT 版本是与开发分支相对应的代码。 用户需要下载源代码并编译相应的jar。 用户应使用已发布的版本，例如 [flink-sql-connector-mongodb-cdc-3.0.0.jar](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-mongodb-cdc), 发布的版本将在 Maven 中央仓库中提供。
+**注意:** 参考 [flink-sql-connector-mongodb-cdc](https://mvnrepository.com/artifact/com.ververica/flink-sql-connector-mongodb-cdc) 当前已发布的版本将在 Maven 中央仓库中提供。
```
Comment (loserwang1024) [inline docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md:52]: missing ','

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md [hunk 3]
```diff
@@ -47,9 +47,9 @@ MongoDB CDC 连接器允许从 MongoDB 读取快照数据和增量数据。 本
 
 ```下载链接仅适用于稳定版本。```
 
-下载 [flink-sql-connector-mongodb-cdc-3.0-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-connector-mongodb-cdc/3.0.0/flink-connector-mongodb-cdc-3.0.0.jar) 把它放在 `<FLINK_HOME>/lib/`.
```
Comment (loserwang1024) [inline docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md:50]: missing ',' or 'and'.

**Review by loserwang1024 — COMMENTED** [review]
I have left some comment.

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md (hunk 3 above)
Comment (lvyanquan) [reply docs/content.zh/docs/connectors/legacy-flink-cdc-sources/mongodb-cdc.md:50]: done.

File: docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md (hunk 1 above)
Comment (lvyanquan) [reply docs/content.zh/docs/connectors/legacy-flink-cdc-sources/db2-cdc.md]: addressed it.

**Review by loserwang1024 — APPROVED** [review]
LGTM!

**Review by leonardBang — APPROVED** [review]
Thanks @lvyanquan for the contribution and @loserwang1024 for the review, LGTM

**Comment by leonardBang** [comment]
@lvyanquan Could you back port the fix to rlease-3.0 branch ?

**Pull request merged**